docs/services/data/_index.md

```diff
@@ -25,11 +25,11 @@ images: ["/services/icons/data-management.svg"]
 no_service: true
 ---
 
-The data management service is a robust solution for smart machine data handling, [data capture](/services/data/capture/), and [cloud synchronization](/services/data/cloud-sync/).
+The data management service is a robust solution for handling machine data, allowing you to capture data, sync it to the cloud, and query it.
 Using the data management service, you can collect data from different parts of a robot, IoT device, or any other machine.
```

Comment:
Hmm I see your point but I believe the point here was not so much about capturing, but rather to emphasize that data could come from any sort of device. Can shorten for brevity, but I don't want to spend too much time reworking this one sentence, since this PR is more about reorganizing the whole data docs structure, unless you have a better suggestion off hand?
```suggestion
You can collect data from different parts of a robot, IoT device, or any other machine.
```

Reply:
i like this "You can.." change, especially in context of our discussion above re the opening sentence not including the user.  Now it's like:

- This is what Data Management is.
- This is how _you_ can use it.

Reply:
Okay I'll apply this and delete the following sentence!